mysql-test/suite/rocksdb_clone/t/ddse_debug.test

```diff
@@ -0,0 +1,47 @@
+--source include/have_rocksdb.inc
```

Comment:
Since the test has remote clone-specific cnf file, suggest to rename to remote_ddse_debug.

Otherwise LGTM!